File: example-projects/junit5-report-portal/src/test/java/com/tngtech/jgiven/example/projects/junit5/JGivenTestConfiguration.java
package com.tngtech.jgiven.example.projects.junit5;

import org.junit.jupiter.api.Tag;

import com.tngtech.jgiven.config.AbstractJGivenConfiguration;

public class JGivenTestConfiguration extends AbstractJGivenConfiguration {

    @Override
    public void configure() {
        configureTag( Tag.class )
            .prependType( true )
            .color( "orange" );
    }

}

File: example-projects/kotlin/src/test/kotlin/com/tngtech/jgiven/example/projects/junit5/JGivenTestConfiguration.kt
package com.tngtech.jgiven.example.projects.junit5

import org.junit.jupiter.api.Tag

import com.tngtech.jgiven.config.AbstractJGivenConfiguration

class JGivenTestConfiguration : AbstractJGivenConfiguration() {

    override fun configure() {
        configureTag(Tag::class.java)
            .prependType(true)
            .color("orange")
    }
}
